### PR TITLE
fix: informative message for agent when permission isn't granted

### PR DESCRIPTION
**Pull request by brayo-pip**

### CODE REVIEW

**Comment by 0xbrayo** [comment]
Helps prevent the loops of the LLM trying to rerun the same command over and over.

File: gptme/tools/shell.py [hunk 1]
```diff
@@ -250,7 +250,7 @@ def execute_shell(
 
     print_preview(cmd, "bash")
     if not confirm("Run command?"):
-        yield Message("system", "Command not run")
+        yield Message("system", "User cancelled command.")
```
Comment (ErikBjare) [inline gptme/tools/shell.py]: I think this is more correct:

```suggestion
        yield Message("system", "User chose not to run command")
```

File: gptme/prompts.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ def prompt_gptme(interactive: bool) -> Generator[Message, None, None]:
     interactive_prompt = """
 You are in interactive mode. The user is available to provide feedback.
 You should show the user how you can use your tools to write code, interact with the terminal, and access the internet.
-The user can execute the suggested commands so that you see their output.
+The user can execute the suggested commands so that you see their output. If user cancels the execution, suggest a different approach.
```
Comment (ErikBjare) [inline gptme/prompts.py]: Not sure if this is needed with my other proposed change.

**Review by ErikBjare — COMMENTED** [review]
Nice find! This was clearly bad prompting before, I can see why it would start looping...

File: gptme/prompts.py (hunk 2 above)
Comment (0xbrayo) [reply gptme/prompts.py]: Will test and see.

Comment (0xbrayo) [reply gptme/prompts.py]: You are right dropped this commit

**Comment by 0xbrayo** [comment]
This is creating merge commits :(

**Pull request closed without merging**